### ISSUE TITLE
No Sphinx documentation for command-line options

### ISSUE THREAD

**Issue by athomps** [FitSNAP/FitSNAP#169]
Apart from the terse info that is provided by the `--help` command-line option, there is no documentation for the many command line options list here. At a minimum, the output from `--help` should be pasted to an *.rst file.

```
usage: FitSNAP3 [-h] [--verbose] [--lammpslog] [--printlammps] [--relative]
                [--nofit] [--overwrite] [--lammps_noexceptions]
                [--keyword GROUP NAME VALUE] [--screen] [--nscreen]
                [--pscreen] [--log LOG] [--screen2file SCREEN2FILE]
                infile

positional arguments:
  infile                Input file with bispectrum etc. options

optional arguments:
  -h, --help            show this help message and exit
  --verbose, -v         Show more detailed information about processing
  --lammpslog, -l       Write logs from LAMMPS. Logs will appear in current
                        working directory.
  --printlammps, -pl    Print all lammps commands
  --relative, -r        Put output files in the directory of INFILE. If this
                        flag is not not present, the files are stored in the
                        current working directory.
  --nofit, -nf          Don't perform fit, just compute bispectrum data.
  --overwrite           Allow overwriting existing files
  --lammps_noexceptions
                        Allow LAMMPS compiled without C++ exceptions handling
                        enabled
  --keyword GROUP NAME VALUE, -k GROUP NAME VALUE
                        Replace or add input keyword group GROUP, key NAME,
                        with value VALUE. Type carefully; a misspelled key
                        name or value may be silently ignored.
  --screen, -sc         Print to screen
  --nscreen, -ns        Print each nodes screen
  --pscreen, -ps        Print each processors screen
  --log LOG             Write fitsnap log to this file
  --screen2file SCREEN2FILE, -s2f SCREEN2FILE
                        Print screen to a file
```

**Comment by rohskopf** [comment]
Added documentation for command line options here: https://fitsnap.github.io/Run/Run_options.html
We can merge the sphinx docs in https://github.com/FitSNAP/FitSNAP/pull/170

**Comment by rohskopf** [comment]
Fixed in #170
